app/src/main/res/values/styles.xml

```diff
@@ -1,27 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <style name="app_default_style">
+        <item name="android:textColor">@color/white</item>
+        <item name="android:textSize">@dimen/activity_main_background_text_size</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:gravity">center</item>
+    </style>
 
-    <style name="button">
-        <item name="android:layout_marginTop">@dimen/fragment_margin_top</item>
-        <item name="android:layout_marginBottom">@dimen/fragment_margin_bottom</item>
+    <style name="main_activity_default_text_view" parent="app_default_style">
+        <item name="android:layout_marginTop">@dimen/main_activity_default_text_view</item>
+    </style>
+
+    <style name="main_activity_available_text_view" parent="app_default_style">
+        <item name="android:layout_marginTop">@dimen/activity_main_text_margin_bottom</item>
+        <item name="android:textColor">@color/honey_background_color</item>
+    </style>
+
+    <style name="fragment_text_view" parent="app_default_style">
+        <item name="android:layout_margin">@dimen/fragment_margin_bottom</item>
+    </style>
+
+    <style name="fragment_input_field" parent="app_default_style">
+        <item name="android:ems">@integer/ems_value</item>
+        <item name="android:inputType">number</item>
+    </style>
+
+    <style name="main_activity_button">
+        <item name="android:layout_marginTop">@dimen/activity_button_margin_top</item>
+        <item name="android:layout_marginBottom">@dimen/activity_main_button_margin_top</item>
         <item name="android:background">@color/honey_background_color</item>
         <item name="android:textColor">@color/dark_grey_text_color</item>
         <item name="android:textStyle">bold</item>
+        <item name="android:padding">@dimen/main_activity_button_padding</item>
+    </style>
+
+    <style name="fragment_button" parent="main_activity_button">
+        <item name="android:layout_marginBottom">@dimen/fragment_margin_bottom</item>
+    </style>
+
+    <style name="picker_button" parent="main_activity_button">
+        <item name="android:layout_marginTop">@dimen/fragment_margin_bottom</item>
+        <item name="android:layout_marginBottom">@dimen/fragment_margin_bottom</item>
     </style>
 
-    <style name="text_view">
-        <item name="android:layout_margin">@dimen/activity_main_text_margin</item>
+    <style name="reserver_activity_button" parent="fragment_button">
```

Comment:
do not create a style for only one property

Reply:
fixed